packages/ui/src/components/va-sidebar/hooks/useSidebar.ts

```diff
@@ -0,0 +1,19 @@
+import { provide, inject, getCurrentInstance, computed, ComputedRef } from 'vue'
+
+const key = 'VaSidebar'
+
+export const useSidebar = () => {
+  const { props } = getCurrentInstance()!
+
+  provide(key, {
+    color: computed(() => props.color),
+  })
+}
+
+export const useSidebarItem = () => {
+  const { color } = inject<{ color: ComputedRef<string> }>(key, { color: computed(() => '') })
```

Comment:
Minor.

As I remember, ComputedRef extends Ref, so we can use Ref type here because ComputedRef is the same as Ref. This will make code smaller and simpler.

```suggestion
  const { color } = inject<{ color: Ref<string> }>(key, { color: ref('') })
```